### PR TITLE
Remove blank spaces if exists in behaviour details and add extra line…

### PR DESCRIPTION
**Pull request by mounikahmcts**
Description: 

Behaviour details is provided as example sections by petitioner. As part of mini petition display these details by removing blank spaces if exists and add extra line break for every example section added by petitioner

Fixes : DIV-3795

### CODE REVIEW

File: steps/review-application/ReviewApplication.html [hunk 1]
```diff
@@ -274,12 +274,17 @@ <h2 class="heading-medium">{{ content.reasonForDivorce }}</h2>
...
     <p class="text">{{ content.reasonForDivorceUnreasonableBehaviourStatement }}</p>
     <p class="text">"
-    {% for item in session.originalPetition.reasonForDivorceBehaviourDetails %}
-      {{ item }}
-      {% if not loop.last %}
-       <br>
-      {% endif %}
-    {% endfor %}
+      {% for behaviourDetail in session.originalPetition.reasonForDivorceBehaviourDetails %}
+        {% if behaviourDetail | length > 0 and behaviourDetail !== '\r' %}
```
Comment (alexwakeman) [inline steps/review-application/ReviewApplication.html]: Seems fine but find this `'\r'` check a bit random - what are you trying to do here?

Comment (mounikahmcts) [reply steps/review-application/ReviewApplication.html]: In PFE, when petitioner provides behaviour details it as example sections as shown in attached image. The data is saved in ccd as array of lines not array of example sections but we need to segregate the sections in  dn & aos app by a line break (attached screeshot) and all the extra spaces should be removed.

CCD data will be like below array. Lines in the same section are separated by \r.
"reasonForDivorceBehaviourDetails": [
      "My wife is 1",
      "My wife is 1\r",
      "My wife is 2",
      "My wife is 1\r",
      "My wife\n is 2\r",
      "My wife is 3",
      "My wife is 1\r",
      "My wife is 2\r",
      "My wife is 3\r",
      "My wife is 4",
      "My wife is 1\r",
      "My wife is 2\r",
      "My wife is 3\r",
      "My wife is 4\r",
      "My wife is 5",
      "My wife is 1\r",
      "My wife is 2\r",
      "My wife is 3\r",
      "My wife is 4\r",
      "My wife is 5",
      "\r",
      "",
      "My wife is 6"
]

PFE updated:


DN Mini petition:

Comment (alexwakeman) [reply steps/review-application/ReviewApplication.html]: So are you always sure the carriage return will be `\r` given this is an (old style) Windows carriage return, whereas Unix uses `\n`?

Also I would suggest writing a JS function to handle separating the strings out using array / string built in operations, as this templating is tricky to figure out on initial inspection and harder to maintain / debug.

Something easy like:
```
this.stringArr = this.stringArr.map((entry) => entry.replace(/\/r/, '<br />'));
```
Mark the template output as `safe` so it retains the HTML output properly.

Comment (mounikahmcts) [reply steps/review-application/ReviewApplication.html]: Moved the processing logic to js file. @alexwakeman  Could you please have a look

**Comment by nsingamsetty** [comment]
This is tested and working as expected. please refer DIV-3795 for evidences. Going to merge this PR.

**Pull request merged**